1.8/administration/overlay-networks/index.md

```diff
@@ -41,9 +41,9 @@ The components of the overlay network interact in the following ways:
 
 Navstar maps IPs to names on your overlay network. During DNS lookup for a task’s IP, there are situations when it is unknown to the DNS server if the IP address stored in NetworkInfo is accessible or not. Three entries in Navstar DNS help to remedy this situation:
 
-* **taskname.marathon.agentip.dcos: agentip**: Provides the agent IP address.
-* **taskname.marathon.containerip.dcos: containerip**: Provides the container IP address.
-* **taskname.marathon.autoip.dcos: autoip**: Provides a best guess of a task's IP address.
+* **taskname.marathon.agentip.dcos.thisdcos.directory: agentip**: Provides the agent IP address.
```

Comment:
Also, to my understanding, agentip is usable regardless of whether you are on the overlay or not, so it should be in another section. This area of the docs imply that they're an overlay-only feature.